docs/source/algorithms/CorelliPowderCalibrationCreate-v1.rst

```diff
@@ -0,0 +1,75 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+
+Description
+-----------
+
+Standard powder samples with Bragg scattering rings of accurately known lattice spacings can be used to adjust the
+components of an instrument such that the observed Bragg scattering yield lattice spacings as close as
+possible to the standard reference.
+
+This algorithm adjusts the position of the moderator as well as the position and orientation of each bank
+of detector pixels. The goal of these adjustments is to produce neutron paths and scattering angles that lead
+to optimal comparison between observed and reference Bragg scattering.
+
+The execution workflow is as follows:
+
+.. diagram:: CorelliPowderCalibrationCreate-v1_wkflw.dot
+
+
+Usage
+-----
+
+* Example -  adjust banks 42 and 87
+
+.. testcode::
+
+    from mantid.simpleapi import *
+    LoadNexus(Filename='/tmp/CORELLI_124036_banks42_87.nxs', OutputWorkspace='LaB6')
```

Comment:
My understanding is that the sphinx doctest will trigger the block and execute the content as part of the doctest.
If this is the case, perhaps we should ensure that the testing file is indeed copied to `/tmp` before invoking `LoadNexus`.